### DISCUSSION TITLE
Should slice-data respect assetPrefix?

### DISCUSSION THREAD

**Discussion by mudge** [gatsbyjs/gatsby#37143] [Ideas / Feature Requests]
We've just deployed our first use of the Slice API and notice requests being made to `/slice-data/header.json` and `/slice-data/footer.json` on our main domain. This required changes to our load balancer as we expected those files to be served from our dedicated asset host (ala files under `/page-data/`) as defined by the `assetPrefix` configuration variable.

Should slice-data also be served with the asset prefix?

**Comment by mudge** [comment]
Specifically, should https://github.com/gatsbyjs/gatsby/blob/33976b3c4decee3fd43e34b410bac182f3930d7a/packages/gatsby/cache-dir/loader.js#L330 use `__PATH_PREFIX__` ala https://github.com/gatsbyjs/gatsby/blob/33976b3c4decee3fd43e34b410bac182f3930d7a/packages/gatsby/cache-dir/loader.js#L31?

**Reply by pieh** [reply]
Yes, I think you are completely right @mudge - we should use same setup for `slice-data` as one we do for `page-data`. Are you willing to open PR with the fix?

**Reply by mudge** [reply]
Raised a PR at https://github.com/gatsbyjs/gatsby/pull/37202
